app/routes/_gcn.circulars._archive._index/SortSelectorButton.tsx

```diff
@@ -0,0 +1,124 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { useSubmit } from '@remix-run/react'
+import {
+  Button,
+  ButtonGroup,
+  CardBody,
+  Grid,
+  Icon,
+  Radio,
+} from '@trussworks/react-uswds'
+import classNames from 'classnames'
+import type { ChangeEvent } from 'react'
+import { useRef, useState } from 'react'
+
+import DetailsDropdownContent from '~/components/DetailsDropdownContent'
+
+function SortButton({
+  sort,
+  expanded,
+  ...props
+}: {
+  sort?: string
+  expanded?: boolean
+} & Omit<Parameters<typeof ButtonGroup>[0], 'segmented' | 'children'>) {
+  const slimClasses = 'height-4 padding-y-0'
+
+  return (
+    <ButtonGroup type="segmented" {...props}>
+      <Button type="button" className={`${slimClasses} padding-x-2`}>
+        Sorted By {sort === 'relevance' ? 'Relevance' : 'Circular'}
+      </Button>
+      <Button type="button" className={`${slimClasses} padding-x-2`}>
+        {<Icon.FilterList role="presentation" />}
+        {expanded ? (
+          <Icon.ExpandLess role="presentation" />
+        ) : (
+          <Icon.ExpandMore role="presentation" />
+        )}
+      </Button>
+    </ButtonGroup>
+  )
+}
+
+export function SortSelector({
+  form,
+  defaultValue,
+}: {
+  form?: string
+  defaultValue?: string
+}) {
+  const [showContent, setShowContent] = useState(false)
+
+  const sortInputRef = useRef<HTMLInputElement>(null)
+
+  const submit = useSubmit()
+
+  const sortOptions = [
+    { id: 'radio-sort-circularId', value: 'circularID', label: 'Circular' },
+    { id: 'radio-sort-relevance', value: 'relevance', label: 'Relevance' },
+  ]
```

Comment:
Lift this data up to module scope so that both this component and `SortButton` can use it.